internal/agent/hooks/kcrypt_uki.go

```diff
@@ -163,33 +161,16 @@ func (k KcryptUKI) Run(c config.Config, _ v1.Spec) error {
 		return err
 	}
 
-	// Copy logs to persistent partition
-	c.Logger.Debug("Copying logs to persistent partition")
-	err = machine.Mount(constants.PersistentLabel, constants.PersistentDir)
-	if err != nil {
-		c.Logger.Errorf("could not mount persistent partition: %s", err)
-		return nil
-	}
-	varLog := filepath.Join(constants.PersistentDir, ".state", "var-log.bind")
-	// Create the directory on persistent
-	err = fsutils.MkdirAll(c.Fs, varLog, 0755)
-	if err != nil {
-		c.Logger.Errorf("could not create directory on persistent partition: %s", err)
-		return nil
-	}
-	// Copy all current logs to the persistent partition
-	err = internalutils.SyncData(c.Logger, c.Runner, c.Fs, "/var/log/", varLog, []string{}...)
+	c.Logger.Logger.Debug().Msg("Finish KcryptUKI hook")
+	// We now have the partitions unlocked and ready, lets call the other hooks here instead of closing and reopening them each time
```

Comment:
so if I understand correctly, this will happen on every kcrypt run instead of just in the after install one

Reply:
no, the kcrypt_uki hook is only run after the install so it will only run once, after the install is done!

Reply:
i see, a bit confusing 😅 so it's not really after install but a bit before

Reply:
Yes, we had to change it for the after I stall stage to have access to the encrypted partitions